vividus-plugin-web-app/src/main/java/org/vividus/selenium/screenshot/WebAshotFactory.java

```diff
@@ -95,10 +94,15 @@ private DebuggingViewportPastingDecorator decorateWithViewportPasting(ShootingSt
         );
     }
 
-    private ShootingStrategy decorateWithScrollbarHiding(ShootingStrategy strategy,
-            Optional<WebElement> scrollableElement)
+    private ShootingStrategy decorateWithScrollbarHiding(ShootingStrategy strategy, WebScreenshotParameters params)
     {
-        return new ScrollbarHidingDecorator(strategy, scrollableElement, scrollbarHandler);
+        if (params == null)
+        {
+            return new ScrollbarHidingDecorator(strategy, Optional.empty(), scrollbarHandler);
+        }
+        return params.isHideScrollbars()
+                ? new ScrollbarHidingDecorator(strategy, params.getScrollableElement(), scrollbarHandler)
```

Comment:
what if someone set: hideScrollbars = true, scrollableElement = "some_element"? it think there should be an error

Reply:
scrollableElement is used to determine the height of a page under test, so these properties are not conflicting